UnitystationLauncher/ViewModels/ChangeViewModel.cs

```diff
@@ -0,0 +1,17 @@
+using UnitystationLauncher.Models.Api.Changelog;
+
+namespace UnitystationLauncher.ViewModels;
+
+public class ChangeViewModel : ViewModelBase
+{
+    private string Type { get; }
+    private string Description { get; }
+    private string AuthorAndPr { get; }
+
+    public ChangeViewModel(Change change)
+    {
+        Type = $"[{change.Category}]";
+        Description = change.Description ?? "null";
+        AuthorAndPr = $"{change.AuthorUsername ?? "null"} | PR #{change.PrNumber}";
```

Comment:
can the PR be clickable and link to the PR on github?

Reply:
Yeah, should be added now.